### PR TITLE
Refill suicideCoinbase test after fixing a typo Homstead -> Homestead

### LINKED ISSUE #294: Misspellt network name in a filler
In one file, `Homestead` is spellt `Homstead`.
```
src/BlockchainTestsFiller/bcStateTests/suicideCoinbaseFiller.json
37:		    "network" : ["Homstead", "EIP150", "EIP158", "Byzantium"],
```

* [x] make sure `testeth -- --filltests` fail on this filler for the spelling mistake https://github.com/ethereum/cpp-ethereum/pull/4435
* [ ] fix the spelling mistake in this file

### PR DESCRIPTION
**Pull request by pirapira**
Fixes #294.

### CODE REVIEW

File: src/BlockchainTestsFiller/bcStateTests/suicideCoinbaseFiller.json [hunk 1]
```diff
@@ -43,6 +43,36 @@
 				"balance" : "5000000000000420000"
 			    },
 			    "a94f5374fce5edbc8e2a8697c15331677e6ebf0b" : {
+				"balance" : "9999472208",
```
Comment (pirapira) [inline src/BlockchainTestsFiller/bcStateTests/suicideCoinbaseFiller.json:46]: I don't understand why this balance changed.

Comment (winsvega) [reply src/BlockchainTestsFiller/bcStateTests/suicideCoinbaseFiller.json:46]: changed from what?

Comment (pirapira) [reply src/BlockchainTestsFiller/bcStateTests/suicideCoinbaseFiller.json:46]: "9999447208",

Comment (winsvega) [reply src/BlockchainTestsFiller/bcStateTests/suicideCoinbaseFiller.json:46]: so there were no bytecode changes on homestead but gas usage has changed? 
lets see what other clients says about this test

Comment (pirapira) [reply src/BlockchainTestsFiller/bcStateTests/suicideCoinbaseFiller.json:46]: OK.

Comment (winsvega) [reply src/BlockchainTestsFiller/bcStateTests/suicideCoinbaseFiller.json:46]: ah wait. you fixed the cpp client from typo? could it be that some code now works on homestead rules now and thus this test is different?

Comment (pirapira) [reply src/BlockchainTestsFiller/bcStateTests/suicideCoinbaseFiller.json:46]: That shouldn't change the test results.

**Pull request merged**